tests/test_configuration/test.ini

```diff
@@ -11,49 +11,131 @@ port = %(http_port)s
 [app:main]
 use = egg:kinto
 
+# Production settings
```

Comment:
Sorry for the delay, I had troubles going through the pile of Github notifications.

I'll update the documentation and you'll decide if you leave it in the `.tpl` or not :)